openquake/server/templates/engine/index.html

```diff
@@ -60,6 +60,25 @@
               <a href="#" data-calc-id="<%= calc.get('id') %>" is-running="<%= calc.get('is_running') %>" class="btn btn-sm btn-log" style="margin: 2px 0 2px 0;">Console</a>
               <% if (calc.get('is_running') == false) { %>
                 <a href="#" data-calc-id="<%= calc.get('id') %>" class="btn btn-sm btn-danger" style="margin: 2px 0 2px 0;">Remove</a>
+                <!-- confirm_modal -->
+                <div class="modal-backdrop  in hide back_confirmDialog<%= calc.get('id') %>"></div>
```

Comment:
I know it was an error just to signal that the same mistake is from other parts in the code.
Fixed here: https://github.com/gem/oq-engine/pull/2876/commits/7c6c9695fc82cecfe6edce75302b44e33b6ef32e